crates/ruff_linter/src/rules/airflow/rules/removal_in_3.rs

```diff
@@ -71,6 +71,63 @@ impl Violation for Airflow3Removal {
     }
 }
 
+const REMOVED_CONTEXT_KEYS: [&str; 12] = [
+    "conf",
+    "execution_date",
+    "next_ds",
+    "next_ds_nodash",
+    "next_execution_date",
+    "prev_ds",
+    "prev_ds_nodash",
+    "prev_execution_date",
+    "prev_execution_date_success",
+    "tomorrow_ds",
+    "yesterday_ds",
+    "yesterday_ds_nodash",
+];
+
+fn extract_name_from_slice(slice: &Expr) -> Option<String> {
+    match slice {
+        Expr::StringLiteral(ExprStringLiteral { value, .. }) => Some(value.to_string()),
+        _ => None,
+    }
+}
+
+pub(crate) fn removed_context_variable(checker: &mut Checker, expr: &Expr) {
```

Comment:
do we need to make it `pub(crate)` here?

Reply:
Right, it is not needed as it is used internally. Removed it.